blueprints/third-party-solutions/f5-bigip/f5-bigip-ha-active/variables.tf

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "f5_vms_dedicated_config" {
+  description = "The F5 VMs configuration. The map keys are the zones where the VMs are deployed."
+  type = map(object({
+    network_config = object({
+      alias_ip_range_address = string
+      alias_ip_range_name    = string
+      dataplane_address      = optional(string)
+      management_address     = optional(string)
+    })
+    license_key = optional(string, "AAAAA-BBBBB-CCCCC-DDDDD-EEEEEEE")
+  }))
+}
+
+variable "f5_vms_shared_config" {
```

Comment:
this is not used in examples AFAIK, can we add one to show how this work and test it?